app/models/restroom.rb

```diff
@@ -17,14 +17,17 @@ class Restroom < ApplicationRecord
   using: {tsearch: {dictionary: "english"}},
   ignoring: :accents
 
-  validates :name, :street, :city, :state, presence: true
+  if Rails.env != "test"
```

Comment:
This does the trick for testing, but I don't like it. I think I'd rather fix the tests to truncate the db and simulate production better. We also need a test for the validation itself.

Reply:
Would we be able to "simulate production better" by having real database data from production? I would be able to update `db/export.csv` with actual, recent data as it exists in production, if that would help.

Reply:
@DeeDeeG I think that would make sense. I'd like to also work on killing this conditional though.

Reply:
Oh, also FYI. Changing the csv won't fix this issue. There's some sort of hook before the save that tries to geolocate these. I'm not sure what forces it to happen, but it is different than the uniqueness check.